libnetdata/queue/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Queue"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/libnetdata/queue/README.md
+-->
+
+# Queue
+
+Queue is a thread-safe library to handle queue processes with independent objects.
+
+Library includes functions below:
+
+- `Queue* initqueue(int max)`
+- `void freequeue(Queue* q)`
+- `void enqueue(void *q, void* item)`	
+- `void* dequeue(void *q)`
+
```

Comment:
We are listing `4` functions but in the line `29` of this document we have a call for a function that is not listed here, please, complete the list. 

I will give another suggestion, instead a list, use a table with two columns, in the first you use the function name and in the second you give a short description for  it.